src/routes/api/organisationJobs.route.ts

```diff
@@ -0,0 +1,38 @@
+import express from 'express';
+export const router = express.Router({ mergeParams: true });
+import { check, oneOf } from 'express-validator';
+import { db } from '../../database';
+import { authorise, checkInputError, protect } from '../../middleware';
+import { INVALID_FIELD_MSG, NO_FIELD_UPDATED_MSG } from '../../utils';
+
+// Import controllers and initialise controller
+import { OrganisationJobsController } from '../../controllers/organisationJobs';
+const organisationJobsController = new OrganisationJobsController(db.organisationJobs, db.organisations);
```

Comment:
For this, i am thinking that we should go with a new way of structuring the `controller`, can we move the declaration of the singleton in the controller file itself? then we import that singleton here like so:
```suggestion
import { organisationJobsController } from '../../controllers/organisationJobs';
```